.circleci/config.yml

```diff
@@ -11,27 +11,23 @@ workflows:
           context: Danger
       - kitchen/lint:
           name: lint
-      # - kitchen/dokken:
-      #     name: default
-      #     suite: default
-      #     requires: [  danger, lint ]
-      # - kitchen/dokken-single:
-      #     name: debian-9
-      #     suite: default
-      #     platform: debian-9
-      #     requires: [  danger, lint ]
-      # - kitchen/dokken-single:
-      #     name: debian-8
-      #     suite: default
-      #     platform: debian-8
-      #     requires: [  danger, lint ]
-      # - kitchen/dokken-single:
-      #     name: ubuntu-1604
-      #     suite: default
-      #     platform: ubuntu-1604
-      #     requires: [  danger, lint ]
-      # - kitchen/dokken-single:
-      #     name: ubuntu-1804
-      #     suite: default
-      #     platform: ubuntu-1804
-      #     requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: debian-9
+          suite: default
+          platform: debian-9
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
```

Comment:
Could you disable the debian 8 and ubuntu tests for now, The package unifi package has broken dependencies at this moment in time. Add a comment at the top so we know why these are disabled as well, otherwise all good :+1: